intelmq/bots/outputs/udp/output.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+import socket
+import sys
+
+import intelmq.lib.utils as utils
+from intelmq.lib.bot import Bot
+
+
+class UDPBot(Bot):
+
+    def init(self):
+        self.delimiter = self.parameters.field_delimiter
+        self.header = self.parameters.header
+        self.udp_host = self.parameters.ip
+        self.udp_port = self.parameters.port
+        self.upd_address = (self.udp_host, self.udp_port)
+        self.udp = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+        self.raw_field = self.parameters.raw_field
+        self.format = self.parameters.format
+
+    def process(self):
+        event = self.receive_message()
+
+        if event is None:
+            self.acknowledge_message()
+            return
```

Comment:
When I cleaned up the code in certtools/intelmq#494, this block has been removed.